explainer.md

```diff
@@ -18,96 +18,107 @@ permanent "block" policies, or
+    blurred), shown after the user has clicked on the empty area next to the
+    article content. The user finds this prompt interruptive as they had no
+    interest in subscribing to notifications, and they will likely struggle to
+    understand why the prompt was shown to begin with.*
+
+1.  **Location**: In the ideal case above, users will interact with something on
+    a site that triggers a prompt. In less ideal cases, the user might not have
+    interacted with anything at all, or they may have interacted with an element
+    that was unrelated to the request. Given this uncertainty, user agents rely
+    on common placement of the permission prompt, usually in the top-left of the
+    page. Even in the best case, this has the unfortunate effect of shifting the
+    point to which users need to pay attention from the thing they clicked on to
+    some distant part of the user agent's UI (see example 2). User agents could
+    make better decisions and provide better prompts if they could make
+    well-founded assumptions about the nature of the user's interaction in the
+    content area, and the user's current area of focus. In effect, we think
+    there is a benefit to semantic markup for permissions.
+
+    ![](images/image2.png) \
+    *Example 2. An example where the permission prompt is far away from the
+    user's current area of focus. The permission prompt was triggered because
+    the user has just clicked on the crosshair icon in the bottom right, but the
+    prompt is easy to miss since it's on the opposite side of the page.*
+
+1.  **Regret**: Given the challenges of permission annoyance and abuse, it is
+    reasonable for user agents to suppress a site's future requests for the same
+    capability when the first request is blocked. That said, our research shows
+    that users can and do change their minds for good reasons. When they change
+    their mind, the site can no longer offer an interface in web content and the
+    user must search for the appropriate user agent surface. Our research shows
+    that users often fail when trying to do so (see example 3). In these cases,
+    the user agent's desire to protect the user backfires, and makes the user's
+    experience worse as the site will not work as the user wants. User agents
+    can help users recover from a permission regret state if they can make
+    well-founded assumptions about the nature of the user's intent and
+    interaction with web content.
+
+    ![](images/image3.png) \
+    *Example 3. An example where the user previously blocked camera and
+    microphone access, but has now just expressed a strong intention to
+    re-enable them by clicking the unmute buttons. Because the user agent has no
+    insight into this interaction in the content area, it is compelled to
+    respect the user's previous decision. Especially in a stressful scenario
+    such as an important presentation, users will struggle to navigate the
```

Comment:
Right, but even here, this is a design flaw in the browser's UI. Making it easier to access the site settings may elevate some of this.

Reply:
Extracted into https://github.com/WICG/PEPC/issues/8